### PR TITLE
OTP user Not replace "/common" when first landing

### PR DESCRIPTION
**Pull request by wanli117**
OTP user is like MSA user, using MICROSOFT_SERVICE_TENANT_ID: "f8cdef31-a31e-4b4a-93e4-5f571e91255a" as default tenant id when first landing. AAD by pass MSA but not OTP. So we are keep using "/common" to fetch MT token.

### CODE REVIEW

File: lib/adal.js [hunk 1]
```diff
@@ -1222,7 +1223,11 @@ AuthenticationContext.prototype.handleWindowCallback = function (hash) {
 
 AuthenticationContext.prototype._getNavigateUrl = function (responseType, resource) {
     var tenant = 'common';
-    if (this.config.tenant) {
+    /**
+     * OTP user (first landing: tenant id == MICROSOFT_SERVICE_TENANT_ID) : DO NOT replace "/common" at first landing.
+     * Other users: replace "/common" if tenant id exists.
```
Comment (chschafl) [inline lib/adal.js]: nit: remove the / in front of common.

File: lib/adal.js [hunk 2]
```diff
@@ -114,7 +114,8 @@ AuthenticationContext = function (config) {
             3: 'VERBOSE:'
         },
         POPUP_WIDTH: 483,
-        POPUP_HEIGHT: 600
+        POPUP_HEIGHT: 600,
+        MICROSOFT_SERVICE_TENANT_ID: "f8cdef31-a31e-4b4a-93e4-5f571e91255a"
```
Comment (chschafl) [inline lib/adal.js]: MSA_TID?

Comment (wanli117) [reply lib/adal.js]: Changed

File: lib/adal.js (hunk 1 above)
Comment (wanli117) [reply lib/adal.js]: Changed.

**Review by amitapl — APPROVED** [review]

**Pull request merged**